src/mesh.js

```diff
@@ -300,6 +300,169 @@ export default class Mesh {
         self.materialsByIndex = {};
     }
 
+    calculateTangentsAndBitangents() {
```

Comment:
Could you add a simple JSDoc comment as well explaining the new attributes that are added by calling this method? And if you're up to it, maybe an explanation of the use of the tangent and bitangents and an example of their use (even if that example is a link to a good tutorial).

Reply:
Sure, I'll add the description!